src/osgText/Text.cpp

```diff
@@ -57,6 +57,10 @@ Text::Text():
         else if (str=="SIGNED_DISTANCE_FIELD" || str=="SDF") _shaderTechnique = SIGNED_DISTANCE_FIELD;
         else if (str=="NO_TEXT_SHADER" || str=="NONE") _shaderTechnique = NO_TEXT_SHADER;
     }
+    else
```

Comment:
This fix now disables use of GREYSCALE for all places where osg::DisplaySettings::instance()->getTextShaderTechnique() is not set, which is the default.  The default is GREYSCALE as this is best default implementation.   

If you want to locally disable this then one should use the osg::DisplaySettings::instance()->getTextShaderTechnique();  or if the OS doesn't support a particular options such as shaders not being available then this is determination that should be done at runtime once the graphics context is tested.  I haven't looked into this to know how practical it would be.  Right now my preference would be if you want to select something that isn't default then _shaderTechnique default should be set via the osg::DisplaySettings::instance()->getTextShaderTechnique(); value or have the Text objects manually set the technique,

Reply:
Okay, it sounded like in your response that you expected the default value to be NO_SHADER.  

I'll try the DisplaySettings approach, the ctor itself crashes so setting the technique on the Text obj after it is created isn't possible.

Reply:
I call `setTextShaderTechnique` as you suggested from my client code which works for me. I removed commits from this PR associated with text shaders / ES1.